fb-negotiation/systems/__init__.py

```diff
@@ -0,0 +1,34 @@
+# from core.price_tracker import PriceTracker, add_price_tracker_arguments
+from rulebased_system import RulebasedSystem
+from cmd_system import CmdSystem
+
+def add_system_arguments(parser):
+    # add_price_tracker_arguments(parser)
+    parser.add_argument('--mappings', default='.', help='Directory to save mappings/vocab')
+
+def get_system(name, args, schema=None, timed=False):
+    lexicon = None # PriceTracker(args.price_tracker_model)
+    if name == 'rulebased':
+        return RulebasedSystem(lexicon, timed)
+    elif name == 'cmd':
+        return CmdSystem()
+    elif name.startswith('ranker'):
+        # TODO: hack
```

Comment:
Remove 'ranker' stuff. In general, try to clean up stuff not needed in this setting.